src/pages/Examples/TopologyPage/TopologyPage.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import topology from "../../../data/topology.json";
+import { TopologyPageView } from "./TopologyPageView";
+
+export const TopologyPage = () => (
+  <TopologyPageView
+    topology={topology[0].components[0].components[0].properties}
```

Comment:
The sample data would be for 1 single card. 
```suggestion
    topology={topology}
```